docker/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM alpine AS installer
+
+RUN wget https://github.com/deislabs/spiderlightning/releases/download/v0.5.0/slight-linux-x86_64.tar.gz
+RUN mkdir slight
+RUN tar -xf slight-linux-x86_64.tar.gz -C slight
+
+RUN wget https://github.com/WebAssembly/wasi-sdk/releases/download/wasi-sdk-20/wasi-sdk-20.0-linux.tar.gz
+RUN mkdir wasi-sdk
+RUN tar -xf wasi-sdk-20.0-linux.tar.gz -C wasi-sdk
+
+FROM mcr.microsoft.com/dotnet/sdk:8.0-preview-jammy
+
+ENV WASI_SDK_PATH=/wasi-sdk/
+COPY --from=installer /wasi-sdk/wasi-sdk-20.0/ $WASI_SDK_PATH
+
+COPY --from=installer /slight/release/slight /usr/bin/
+
+RUN dotnet workload install wasi-experimental -s https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet8/nuget/v3/index.json
```

Comment:
I *think* the `-s ...` might not be required now preview 4 has shipped, but am not 100% sure.

Reply:
Oh I probably should have waited for your feedback on this before merging. It's not a big deal but I'm happy to change it further if you think it worthwhile.

Reply:
OK I did this to save you from having to do another PR: https://github.com/SteveSandersonMS/spiderlightning-dotnet/commit/2cdae45edeae85d3eac571e8c282422715324c90